src/main/java/org/sasanlabs/service/vulnerability/jwt/JWTVulnerability.java

```diff
@@ -662,4 +669,51 @@ private ResponseEntity<GenericVulnerabilityResponseBean<String>> getJWTResponseB
                         true, token, true, CollectionUtils.toMultiValueMap(headers));
         return responseEntity;
     }
+
+    @AttackVector(
+            vulnerabilityExposed = VulnerabilityType.HEADER_INJECTION,
+            description = "HEADER_INJECTION_VULNERABILITY_EXAMPLE")
+    @VulnerableAppRequestMapping(
+            value = LevelConstants.LEVEL_13,
+            htmlTemplate = "LEVEL_13/HeaderInjection_Level13")
+    public ResponseEntity<GenericVulnerabilityResponseBean<String>> getHeaderInjectionVulnerability(
```

Comment:
Level9 has JWK based vulnerability. How about converting it into a secure level where we always validate JWK header against a set of public keys and only allow if it is part of them ?

Reply:
@leiberbertel it is fine to have another vulnerability with same functionality as well.

Reply:
Sure, I'll work on the implementation

Reply:
I understand, thanks for the confirmation:).